openassessment/xblock/static/sass/oa/views/_oa-base.scss

```diff
@@ -286,7 +286,7 @@
       }
 
       .step__counter:before, .step__label, .step__deadline {
-        color: $color-unavailable;
+        color: $copy-color;
```

Comment:
Step counters before were often a very pale color. Make them the regular, accessible text color.

Before:
![image](https://cloud.githubusercontent.com/assets/484484/20154427/3c3d3050-a695-11e6-8dd0-274a7fa54a64.png)

After:
![image](https://cloud.githubusercontent.com/assets/484484/20154437/461dc1e8-a695-11e6-8ea3-72a1f2a6b175.png)